homeassistant/components/google_assistant/trait.py

```diff
@@ -1073,8 +1073,8 @@ def sync_attributes(self):
 
     def query_attributes(self):
         """Return ArmDisarm query attributes."""
-        if "post_pending_state" in self.state.attributes:
-            armed_state = self.state.attributes["post_pending_state"]
+        if "next_state" in self.state.attributes:
+            armed_state = self.state.attributes["next_state"]
```

Comment:
We shouldn't have added this logic to begin with. It needs to be supported by the core alarm control panel entity model before other integrations can use this logic.

Reply:
When I first added this I was using the manual component. I think we should apply this fix for now until this logic is moved to the core alarm control panel then we can do another PR base that.

Reply:
ok